src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {useEffect} from 'react';
+import Animated, {cancelAnimation, Easing, runOnJS, useAnimatedStyle, useSharedValue, withDelay, withRepeat, withSequence, withTiming} from 'react-native-reanimated';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+function ProgressBar({shouldShow}: {shouldShow: boolean}) {
+    const left = useSharedValue(0);
+    const width = useSharedValue(0);
+    const opacity = useSharedValue(0);
+    const isVisible = useSharedValue(false);
+    const styles = useThemeStyles();
+
+    useEffect(() => {
+        if (shouldShow) {
+            // eslint-disable-next-line react-compiler/react-compiler
+            isVisible.value = true;
+            left.value = 0;
+            width.value = 0;
+            opacity.value = withTiming(1, {duration: CONST.ANIMATED_PROGRESS_BAR_OPACITY_DURATION});
+            left.value = withDelay(
+                CONST.ANIMATED_PROGRESS_BAR_DELAY,
+                withRepeat(
+                    withSequence(
+                        withTiming(0, {duration: 0}),
+                        withTiming(0, {duration: CONST.ANIMATED_PROGRESS_BAR_DURATION, easing: Easing.bezier(0.65, 0, 0.35, 1)}),
+                        withTiming(100, {duration: CONST.ANIMATED_PROGRESS_BAR_DURATION, easing: Easing.bezier(0.65, 0, 0.35, 1)}),
```

Comment:
same here

Reply:
@getusha We didn't add it in the const with the parameters of Easing.bezier like [here](https://github.com/Expensify/App/blob/e52d43c2cd8ae7d8449161651867e8b163ac1321/src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx#L69)

Reply:
friendly bump @getusha